uber/templates/common.js

```diff
@@ -71,14 +71,16 @@ var setupRatingClickHandler = function () {
 };
 var setStatus = function (shiftId, status) {
     var $status = $(status);
-    $.post('../jobs/set_worked', {id: shiftId, worked: $status.val(), csrf_token: csrf_token}, function (result) {
+    var statusVal = parseInt($status.val());
+    $.post('../jobs/set_worked', {id: shiftId, status: statusVal, csrf_token: csrf_token}, function (result) {
         if (result.error) {
             alert(result.error);
         } else {
+            var statusLabel = _(result.shifts).filter({id: shiftId}).pluck('worked_label').first() || 'Unexpected Error';
```

Comment:
Previously we had different ``set_worked`` functions depending on where you were coming from, e.g. ``set_worked`` vs ``set_worked_from_everywhere``.  We condensed them into a single ``set_worked``, but I forgot to update this code to expect to get back a ``job`` object as the result instead of the ``shift`` object we were getting back before.  That was the bug, and this is the fix.